scripts/catalog/json.go

```diff
@@ -28,5 +36,13 @@ func main() {
 		log.Fatal(err)
 	}
 
+	countFile := "internal/provider_count.go"
```

Comment:
As commented in the file, make it to include `generated` to make it easier to understand when importing.